core/pom.xml

```diff
@@ -150,6 +150,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.python</groupId>
+            <artifactId>jython</artifactId>
+            <version>2.7.0</version>
```

Comment:
For a while, we bundled jython (via winrm4j, until we re-wrote that as pure Java). The jython libraries pulled in about 20 or 30MB of dependencies. It would certainly be nice if folk can add that to the classpath (or OSGi bundles) themselves, but I don't think we want to ship it with Brooklyn.